### ISSUE TITLE
instagrapi.exceptions.ClientError:401 Client Error

### ISSUE THREAD

**Issue by goodboyaloy** [adw0rd/instagrapi#993]
Hoping to get some assistance. 

I started getting 401 errors while scraping with without logging in. I'm only sending 1 request.

`requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://www.instagram.com/nike/?__a=1&__d=dis`

What I have tried:
1.  Using rotating residential IPs
2. Scraping without proxies
3. Connecting to a different network

It seems that I can make these requests through my browser with no problem, but with the library, I get 401 errors 
(with or without proxies).

I tried to search up about this but couldn't find anything helpful - I was hoping someone here has experience dealing with this. Thank you!

**Comment by 07h** [comment]
Apparently, the developer has stopped working on the library. Instagram easily detects work through instagrapi

**Comment by adw0rd** [comment]
@07h everything works fine, I make 2-3 million requests per day through instagrapi, use clean proxies or log in to instagram

**Comment by Yan19960220** [comment]
I got the same error when I successfully login in many times.

**Comment by artgadliba** [comment]
> @07h everything works fine, I make 2-3 million requests per day through instagrapi, use clean proxies or log in to instagram

I dont want to spawn the same topics, so I decided to post in a closed one. I faced the same issue. I am logged in, as well as settings was saved, so I can send any kind of private requests one by one. But once I am trying to send public GQL request, I got this error: requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url. 

Any small hint would be highly appreciated.
